src/eventCommandHandlers/give/giveCommandHandler.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { expect } from 'chai'
+import GiveCommandHandler from './giveCommandHandler'
+import { ISlackEventInfo } from '../interfaces'
+
+const slackEventInfoFromUserWithFullCommand: ISlackEventInfo = {
```

Comment:
We could make some common function to create test objects/mocks.
```suggestion
import merge from 'lodash/merge'

const someBaseObject = { .... }

const createTestObject = (additionalProperties: Partial<ISlackEventInfo>) => merge(someBaseObject, additionalProperties)
```